app/code/Magento/Sales/Helper/Invoice/Logo.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Magento\Sales\Helper\Invoice;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\UrlInterface;
+use Magento\Store\Model\ScopeInterface;
+
+class Logo extends AbstractHelper
+{
+    const XML_PATH_SALES_IDENTITY_LOGO_HTML = 'sales/identity/logo_html';
+
+    protected $logoBaseDir = 'sales/store/logo_html/';
+
+    /**
+     * @return string|null
+     */
+    public function getLogoFile()
+    {
+        $result = null;
+
+        $invoiceLogoPath = $this->getIdentityLogoHtml();
+        if ($invoiceLogoPath) {
+            $result = $this->_urlBuilder->getBaseUrl(
+                ['_type' => UrlInterface::URL_TYPE_MEDIA]
+            ) . $this->getLogoBaseDir() . $invoiceLogoPath;
+        }
+
+        return $result;
+    }
+
+    /**
+     * @return string
+     */
+    public function getLogoBaseDir()
```

Comment:
Please make this method private, declare a method return type, and add a comment to dockblock.